conf/modules.config

```diff
@@ -379,6 +381,10 @@ process {
         if(System.getenv('GITHUB_ACTION') != null ) {
             container = "ghcr.io/sanger-tol/longranger:2.2.2-c3"
         }
+        else {
+            envWhitelist = "LSF_BINDIR,LSF_SERVERDIR,LSF_LIBDIR,LSF_ENVDIR"
+            runOptions = "-B ${projectDir}/martian.lsf.template:/opt/longranger-2.2.2/martian-cs/2.3.2/jobmanagers/lsf.template -B /software -B /etc/lsf.conf --env APPEND_PATH=$LSF_BINDIR:$LSF_SERVERDIR:/software/singularity-v3.9.0/bin"
```

Comment:
@muffato Does this make sense? Is this too Sanger specific to be here?

Reply:
Fixed in https://github.com/sanger-tol/genomeassembly/pull/19